soql-stdlib/src/main/scala/com/socrata/soql/functions/SoQLTypeInfo.scala

```diff
@@ -1,5 +1,6 @@
 package com.socrata.soql.functions
 
+import com.socrata.soql.ast.Hole
```

Comment:
Not yet used?

Reply:
Leftover from a time when the holes were typed.